### PR TITLE
Add completed boolean to Todo object in graphql schema

### PR DESCRIPTION
**Pull request by auroq**

### CODE REVIEW

File: todo-backend/go.mod [hunk 1]
```diff
@@ -5,17 +5,20 @@ go 1.17
 require (
 	github.com/99designs/gqlgen v0.14.0
 	github.com/google/uuid v1.3.0
+	github.com/stretchr/testify v1.7.0
```
Comment (danethurber) [inline todo-backend/go.mod:8]: ah shit, you caved

**Review by danethurber — APPROVED** [review]
👍

File: todo-backend/graph/schema.graphqls [hunk 2]
```diff
@@ -11,6 +11,7 @@ type PageInfo {
 type Todo {
     id: ID!
     title: String!
+    complete: Boolean!
```
Comment (danethurber) [inline todo-backend/graph/schema.graphqls:14]: did you want to name it `completed`?

File: todo-backend/go.mod (hunk 1 above)
Comment (auroq) [reply todo-backend/go.mod:8]: I did. Let's see how we like it.

File: todo-backend/graph/schema.graphqls (hunk 2 above)
Comment (auroq) [reply todo-backend/graph/schema.graphqls:14]: sounds good to me

**Pull request merged**